.vib/postgresql/vib-publish.json

```
@@ -35,16 +35,25 @@
           "action_id": "trivy",
           "params": {
             "threshold": "CRITICAL",
-            "vuln_type": [
-              "OS"
-            ]
+            "vuln_type": ["OS"]
           }
         },
         {
           "action_id": "health-check",
           "params": {
             "endpoint": "lb-postgresql-tcp-postgresql"
           }
+        },
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "goss"
```

I think you missed this.
```suggestion
              "path": "/.vib/postgresql/goss"
```

---

This is a good catch, the pipeline would fail if this mistake hasn´t been fixed.